controller/RestEligibilities.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+namespace oat\taoTestCenter\controller;
+
+use common_exception_MissingParameter;
+use common_exception_NotFound;
+use common_exception_RestApi;
+use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use oat\taoTestCenter\model\eligibility\Eligibility;
+use oat\taoTestCenter\model\EligibilityService;
+
+class RestEligibilities extends AbstractRestController
+{
+
+    const PARAMETER_DELIVERY_ID = 'delivery';
+
+    /**
+     * @throws \common_exception_NotImplemented
+     */
+    public function post()
+    {
+        $this->returnFailure(new common_exception_RestApi('Not implemented.'));
+    }
+
+    /**
+     * @OA\Get(
+     *     path="/taoTestCenter/api/eligibilities",
```

Comment:
I think it's not straightforward that we have and endpoint which indicates that the result will be multiple resources, but it can return only one resource. I would say this logic should be done in `/taoTestCenter/api/eligibility`. What do you think?

Reply:
We can not easily implement it in this endpoint because all GET requests are routed to get() method in controller and it's already used to get eligibility by id. I will return array of resources here. I future we need to improve this endpoint to perform real search by one of parameters with pagination and etc.

Reply:
It's better now :) Could you please update the the swagger documentations as well?